cloudformation.yml

```
@@ -0,0 +1,18 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  MyEC2Instance:
+    Type: "AWS::EC2::Instance"
+    Properties:
+      InstanceType: "t2.micro"
+      KeyName: "MyKeyPair" # Potential Issue: Hardcoded sensitive value
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)